x-pack/plugins/task_manager/server/lib/create_managed_configuration.ts

```diff
@@ -80,7 +81,7 @@ function createMaxWorkersScan(logger: Logger, startingMaxWorkers: number) {
       );
       if (previousMaxWorkers === startingMaxWorkers) {
         logger.warn(
-          `Max workers configuration is temporarily reduced after Elasticsearch returned ${errorCount} "too many request" error(s).`
+          `Max workers configuration is temporarily reduced after Elasticsearch returned ${errorCount} error(s).`
```

Comment:
How do you think this reasons should looks like if we have both type of errors - comma separated? I'm not sure that we really need this, because the errors were logged by itself.

Reply:
I believe the "too many request" errors are not logged elsewhere and mostly accumulated/ignored. Maybe something like `"too many request" and/or "execute [inline] script"`.